CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @AngeloDanducci
+*       @scottlinder
```

Comment:
Not sure we can trust, Scott, he works for AMD now ...

Reply:
Well, we could leave him on a probationary period. ;)